backend/routes/login.js

```diff
@@ -0,0 +1,16 @@
+const Router = require('express');
+const { body } = require('express-validator');
+const { validateLogin } = require('../handlers/login');
+const { handleInputErrors } = require('../modules/middleware');
+
+const loginRouter = Router();
+
+loginRouter.post(
+  '/login',
+  body('username').isString(),
```

Comment:
consider converting all of these validator chains into schemas instead: https://express-validator.github.io/docs/api/check-schema#schema

IMO it's a little more easy to reason about and share/recycle future request schemas

Reply:
Done!